.vib/jaeger/cypress/cypress/integration/jaeger_spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+it('list and retrieve jaeger traces', () => {
+
+    const testService = 'redis';
+    const currentDate = new Date(); 
+    const timestampMillis = currentDate. getTime() * 1000;
+    // Check application availability though the UI
+    cy.visit('/search?end=' + timestampMillis + '&limit=20&lookback=1h&maxDuration&minDuration&service=' + testService + '&start=0').then(() => {
+
+        // Ensure page contains Traces in an H2 tag
+        cy.contains('h2', 'Traces');
+
+        let traceIDLink = '';
+        // Get <a> tag link of class "ResultItemTitle--item ub-flex-auto"
+        cy.get('a.ResultItemTitle--item.ub-flex-auto').then((href) => {
```

Comment:
The selector you are using does not seem to be resilient and may change, which may lead to test failures in the future. Alternatively, you can use something like the following:

```suggestion
        cy.contains('a', '/dispatch').then((href) => {
```

Reply:
Done